metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -191,23 +216,6 @@ final class Doctor(
 
   def allTargets(): List[ScalaTarget] = buildTargets.all.toList
 
-  private def problemSummary: Option[String] = {
```

Comment:
Changed to just print unsupported versions if there is an issue, since semanticDB is added automatically by Bloop, so if there is a problem then it means user is using an unsupported version.

Reply:
Actually, it still make sense to print warning about lacking semanticDB in case of using other BSP servers. Did a fallback, since I think reporting unsupported Scala versions is more important.